src/vulkan/pipeline.cc

```diff
@@ -57,6 +57,7 @@ Result Pipeline::InitializeCommandBuffer(VkCommandPool pool, VkQueue queue) {
   if (!r.IsSuccess())
     return r;
 
+  // TODO(jaebaek): create pipeline_cache_
```

Comment:
Sorry, I thought I added an issue for this.
#131 is related issue.
I think this is not a high priority task now because it is related to performance.

Reply:
Let me remove comments related to pipeline cache